torch/_torch_docs.py

```diff
@@ -5657,6 +5657,11 @@ def merge_dicts(*dicts):
 Computing the Matrix Exponential with an Optimized Taylor Polynomial Approximation.
 Mathematics 2019, 7, 1174.
 
+.. seealso::
+
+        :func:`~torch.nn.utils.parametrizations.orthogonal` for a parametrization that makes a
```

Comment:
I don't think we usually do this.
While I agree that it help discoverability in this case, I'm not sure this would scale if we start adding a lot of these.

cc @mruberry